rosidl_typesupport_tests/test/rosidl_typesupport_cpp/test_service_typesupport.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright 2022 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+// #include <rosidl_runtime_cpp/message_type_support_decl.hpp>
+#include "rosidl_runtime_cpp/service_type_support_decl.hpp"
+#include "rosidl_runtime_cpp/message_type_support_decl.hpp"
+#include "rcutils/allocator.h"
+
+#include "rosidl_typesupport_cpp/service_type_support.hpp"
+
+#include "rosidl_typesupport_tests/srv/basic_types.hpp"
+
+#include "builtin_interfaces/msg/time.h"
+#include "unique_identifier_msgs/msg/uuid.h"
+#include "service_msgs/msg/service_event_info.h"
+#include "rosidl_runtime_c/service_type_support_struct.h"
+
+TEST(test_service_typesupport, basic_types_cpp_to_cpp)
+{
+  rcutils_allocator_t allocator = rcutils_get_default_allocator();
+  const rosidl_service_type_support_t * srv_ts = 
+    rosidl_runtime_cpp::get_service_type_support_handle<rosidl_typesupport_tests::srv::BasicTypes>();
```

Comment:
I guess this should use a different namespace:
```suggestion
    rosidl_typesupport_cpp::get_service_type_support_handle<rosidl_typesupport_tests::srv::BasicTypes>();
```
I don't know _why_, but it seems to fix the linker issue.

Reply:
Fixed in [62b9a1d](https://github.com/ros2/rosidl_typesupport/pull/127/commits/62b9a1d61d53afea91b5eb4b11d35baef19ae56f)